tensorhive/mailbot_config.ini

```diff
@@ -0,0 +1,58 @@
+# This config file is interpreted only with these settings:
+# (main_config.ini)
+# [protection_service]
+# notify_via_email = yes
+
+[general]
+# Mail only once every X minutes
+interval = 10.0
+notify_intruder = yes
+notify_admin = no
+;admin_email = tensorhive.mailbot@gmail.com
+
+# Email account configuration for mailbot (required)
+[smtp]
+email_env_var = TENSORHIVE_MAILBOT_EMAIL
+password_env_var = TENSORHIVE_MAILBOT_PASSWORD
+smtp_server = smtp.gmail.com
```

Comment:
There should be only a description "mail server hostname", instead of a specific address

Reply:
![image](https://user-images.githubusercontent.com/12485656/52073710-78901800-2588-11e9-86c3-3866a6c0066c.png)
`mailbot_config.ini`
![image](https://user-images.githubusercontent.com/12485656/52073717-7ded6280-2588-11e9-9a4e-99c8edff7e75.png)
`MAILBOT` config class:
![image](https://user-images.githubusercontent.com/12485656/52073796-b725d280-2588-11e9-9174-57519cd6ace4.png)

Reply:
@roscisz On our last meeting (31.01.2019) you suggested that admin should get emails less frequently than intruder (e.g. just once every `X` intruder's notifications, or once per unprivileged process, etc.)

I think it'll be better to **introduce this feature later** (with bootstrap or something).
I created more detailed issue for this problem: https://github.com/roscisz/TensorHive/issues/145